requirements/dev.in

```diff
@@ -1,37 +1,18 @@
--r base.txt
-
-pip-tools  # https://github.com/jazzband/pip-tools/
+-r test.txt
 
 # Utils
 # ------------------------------------------------------------------------------
-beautifulsoup4  # https://code.launchpad.net/beautifulsoup
 ipdb  # https://github.com/gotcha/ipdb
 
 # Code quality
 # ------------------------------------------------------------------------------
-black  # https://github.com/psf/black
 coverage  # https://github.com/nedbat/coveragepy
-djlint  # https://www.djlint.com
 pre-commit  # https://github.com/pre-commit/pre-commit
 pylint  # https://github.com/PyCQA/pylint
 pylint-django  # https://github.com/PyCQA/pylint-django
```

Comment:
Ca pose vraiment la question de les conserver. Pylint ne passe pas et n'est pas vérifié, et maintenant que les linters sont dans `test.in`...

Reply:
Meme remarque pour coverage.

Reply:
Je suis aussi de cet avis. Je vais proposer le retrait de dépendances qui me semblent discutables dans une autre PR :)

Reply:
https://github.com/betagouv/itou/pull/2775